# HG changeset patch
# User "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
# Date 1322160257 -3600
# Node ID ac0db1b6e9d7accf7353d8ea4589cf212c7005c1
# Parent  3722437d10a241c51d3b9cf9dd429047e0c03bcc
arch/powerpc: PowerPC supports setting the floating point type

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>

diff --git a/config/arch/powerpc.in b/config/arch/powerpc.in
--- a/config/arch/powerpc.in
+++ b/config/arch/powerpc.in
@@ -7,6 +7,7 @@
 ## select ARCH_SUPPORTS_WITH_ABI
 ## select ARCH_SUPPORTS_WITH_CPU
 ## select ARCH_SUPPORTS_WITH_TUNE
+## select ARCH_SUPPORTS_WITH_FLOAT
 ##
 ## help The PowerPC architecture, as defined by:
 ## help     http://www.ibm.com/developerworks/eserver/articles/archguide.html
